javaagent-muzzle-generation/build.gradle.kts

```diff
@@ -0,0 +1,22 @@
+plugins {
```

Comment:
Do we want the word muzzle in these projects? Assuming muzzle refers to our safety mechanism for applying instrumentation across versions, a lot of the codegen is independent of muzzle

Reply:
This plugin generates bodies for `InstrumentationModule` methods with word `muzzle` in method names. So currently it is certainly muzzle-related.